lib/ansible/utils/module_docs_fragments/url.py

```diff
@@ -61,4 +60,11 @@ class ModuleDocFragment(object):
       - PEM formatted file that contains your private key to be used for SSL client authentication.
       - If C(client_cert) contains both the certificate and key, this option is not required.
     type: path
+  tls_insecure:
+    description:
+      - Enable insecure TLS protocols.
+      - This may be required to support older devices.
```

Comment:
I think this (and the other module docs for `tls_insecure`) should contain a warning that this option should only be used if really needed, because it is insecure. (Cannot be mentioned often enough. Otherwise it might end up as a "magic solution" similarly to setting `validate_certs: no` everywhere, which "solves" a lot of problems and introduces a lot of security holes.)

Reply:
I added a statement similar to what is used for `validate_certs`. However, enabling `tls_insecure` doesn't disable secure TLS options, it simply allows for falling back to insecure (older) TLS options if more secure alternatives are not available.